test/buildah-bud/apply-podman-deltas

```diff
@@ -245,7 +245,8 @@ skip_if_remote "Explicit request in buildah PR 4190 to skip this on remote" \
 # BEGIN tests which are skipped due to actual podman or podman-remote bugs.
 
 skip_if_remote "different error messages between podman & podman-remote" \
-               "bud with .dockerignore #2"
+               "bud with .dockerignore #2" \
+               "bud with .dockerignore #4"
```

Comment:
The bud test seems to be a hard failure. I would just add one more line here, `"bud with .dockerignore #6"`